runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/SimpleParDoFn.java

```diff
@@ -361,6 +361,7 @@ private void processUserTimer(TimerData timer) throws Exception {
       fnRunner.onTimer(
           timer.getTimerId(),
           timer.getTimerFamilyId(),
+          this.stepContext.stateInternals().getKey(),
```

Comment:
you'll need the change to StreamingDataflowWorker as well, without it, the key will be null.

Reply:
@steveniemitz added.